guide/twilio_sample.bal

```diff
@@ -17,148 +17,116 @@
 // This guide is created for the reference of "How to Extend Ballerina" in ballerina.io
 // https://ballerina.io/learn/how-to-extend-ballerina/
 
-import ballerina/http;
-import ballerina/io;
-
-// Endpoint
-documentation {Object for Twilio endpoint.
-    F{{twilioConfig}} Reference to TwilioBasicConfiguration type
-    F{{twilioConnector}} Reference to TwilioConnector type
-}
-public type TwilioClient object {
 
-    public TwilioConfiguration twilioConfig;
-    public TwilioConnector twilioConnector = new;
+import ballerina/http;
+import ballerina/mime;
+
+// Twilio API urls
+final string TWILIO_API_BASE_URL = "https://api.twilio.com/2010-04-01";
```

Comment:
Purpose of this file is to aggregate all necessary bits so that how-to-extend-ballerina tutorial follower can read it easily without having to navigate multiple files. Hence all the code here are direct copy-paste from original source.

Reply:
Got it. But it is better if we create an issue to track this change (later) the original code and the tutorial as well.